### PR TITLE
change minion to node

### PR DESCRIPTION
**Pull request by zhengguoyong**
change minion to node

### CODE REVIEW

File: cmd/integration/integration.go [hunk 1]
```diff
@@ -280,7 +280,7 @@ func makeTempDirOrDie(prefix string, baseDir string) string {
 	return tempDir
 }
 
-// podsOnMinions returns true when all of the selected pods exist on a minion.
+// podsOnMinions returns true when all of the selected pods exist on a node.
```
Comment (pmorie) [inline cmd/integration/integration.go]: @zhengguoyong not going to rename this method? :)

Comment (zhengguoyong) [inline cmd/integration/integration.go]: @pmorie 
done.
thanks so much for your review, please check again :)

**Comment by brendandburns** [comment]
LGTM.

**Comment by brendandburns** [comment]
@zhengguoyong needs a rebase (sorry)

**Comment by zhengguoyong** [comment]
@brendandburns 
ok, i rebased it. please check again.
thanks very much.

**Comment by ashcrow** [comment]
The tests seemed to overall pass:

```
Ran 84 of 174 Specs in 663.666 seconds
SUCCESS! -- 84 Passed | 0 Failed | 2 Pending | 88 Skipped 
```

But there is a lot of errors in generating documentation and boilderplate (for items in _gopath ...).

Shippable 1.3 shows 1 failure where Jenkins e2e doesn't:

```
FAIL	k8s.io/kubernetes/pkg/kubelet/network/cni	0.099s
```

Shippable 1.4 passed with no errors.

These results don't make a lot of sense. You may want to rekick the checks. :smile:

**Comment by brendandburns** [comment]
@k8s-bot test this please

**Comment by brendandburns** [comment]
Kicked the tests.

**Comment by a-robinson** [comment]
This is going to need a rebase before it can be merged

**Comment by ashcrow** [comment]
@zhengguoyong looks like this needs another rebase.

**Comment by zhengguoyong** [comment]
sorry for my late reply,
i have rebased again. thanks.
@ashcrow

**Comment by piosz** [comment]
Removing lgtm since there was a change after lgtm was granted. @brendandburns could you please take a look?

**Comment by zhengguoyong** [comment]
hi brendandburns,
would you please make a review for my this PR. thanks.
@brendandburns

**Comment by piosz** [comment]
LGTM

**Pull request merged**